docs/tutorials/kubernetes/ovhcloud.md

```diff
@@ -0,0 +1,452 @@
+# Scan for vulnerabilities and misconfigurations of your OVHcloud Managed Kubernetes with Trivy
```

Comment:
Are we sure this is a "first party" tutorial and not "third party"? do we want to maintain it?